### ISSUE TITLE
LOCALISATION REQUEST: Gronings

### ISSUE THREAD

**Issue by wieling** [common-voice/common-voice#4424]
# Welcome to the Common Voice Community !

> Common Voice aims to make speech technology accessible to everyone by building an open sourced dataset of labelled voice data that is representative of languages, variants and accents spoken across the world. This template helps us to know how your language could participate in the Common Voice Project. There are three sections of this form, once you have filled out a section please click the checkbox. If you have any issues please contact <commonvoice@mozilla.org>.

## Pontoon Set-up

> To start a language on Common Voice volunteers localise our platform [via Pontoon](https://pontoon.mozilla.org/projects/common-voice/) and create sentence corpus’ of cc0 text. </p>

**Language name**

Gronings

**Language code**

gos 
(ISO 639-3)

**Language size**

About 200,000 estimated.

**Plural forms**
0 stainen
1 stain
2 stainen 
3 stainen
4 stainen
5 stainen
10 stainen
20 stainen
100 stainen
1000 stainen
Ik zai 0 stainen op grond
Ik zai 1 stain op grond
Ik zai 10 stainen op grond
Ik zai stainen op grond

**Pontoon manager**

https://pontoon.mozilla.org/contributors/0ofHVNKn6_yGdRZPr8_-d6dR7qY/

**Language Script**

latin

## Sentence Collection Requirements

> On the Common Voice Platform contributors on the platform read out public domain sentences generated through sentence collection. Sentence collection is a crucial part in launching languages on Common Voice. To support the equitable participation of languages of Common Voice we have introduced [three new sentence collection requirements bands](https://discourse.mozilla.org/t/share-your-views-nuancing-sentence-collection-requirements-new-sentence-collection-bands/93134).

**Sentence Requirement Band**

- [x] Band A

- [ ] Band B

- [ ] Band C

## Creating Community 

1. [optional to share] Why do you want to take part in Common Voice ? Enable centralized data collection for improving speech recognition performance for Gronings.

3. [optional to share] Would you like to have a follow up conversation regarding community building ? No.

**Comment by ftyers** [comment]
Hi @wieling thanks for your interest in Common Voice! Could you let me know what the ISO-639-2/3 language code is for Gronings and what the population size is (you can edit your post). I suppose the script is Latin (given the translations). Any questions feel free to get in contact with us on Matrix: https://app.element.io/#/room/#common-voice:mozilla.org

**Comment by wieling** [comment]
Sorry, I apparently submitted while still editing the post... Now all information should be present. We have a large corpus of Groningen texts, so I should be able to create the necessary sentences for reading. Can you let me know at what stage this is necessary? This is not entirely clear to me.

**Comment by wieling** [comment]
I'm also not entirely clear about which checkboxes I need to check?

**Comment by ftyers** [comment]
Probably Band B would suit Gronings best. In terms of the sentences, you can add them either via the [bulk upload process](https://github.com/common-voice/common-voice/blob/main/docs/submitting-bulk-sentences.md) or via the interface on the main site, once you have reached 70% translation. 

Common Voice has been enabled for Gronings for translation, you can access it [here](https://pontoon.mozilla.org/gos/common-voice/). Please make a couple of suggestions and I will add you as a translator.

**Comment by wieling** [comment]
Is it necessary to provide a translation of the interface in Gronings? All speakers of Gronings can use Dutch, so if possible, we'd prefer using that language for the interface itself. Is that possible?

**Comment by ftyers** [comment]
Unfortunately no, the current policy of Common Voice is to have the interface localised in the language which will be used for voice collection.

**Comment by wieling** [comment]
OK, clear. Can we also get access to the Dutch translations (and how?)? As that will be easier for those doing the translations.

**Comment by ftyers** [comment]
@wieling  Yes, each person can change their preferred language in their Pontoon profile.

![imatge](https://github.com/common-voice/common-voice/assets/449545/d9e59691-1f57-4d3d-94d9-5d8c01c6eaf4)

See "Default locale -> Preferred source locale".

**Comment by wieling** [comment]
Thanks! One remaining question: is it also possible to batch update all translations (and download them in the source language)?

**Comment by ftyers** [comment]
@wieling were you able to make a translation suggestion?

**Comment by wieling** [comment]
Not yet, sorry. Due to the CC0 requirement, we might instead opt to host an
implementation at our university instead, so we can use the available texts
(and don't need to translate the interface).

**Comment by ftyers** [comment]
Hi @wieling, just a heads up that there has been movement on the [Gronings Common Voice translation](https://pontoon.mozilla.org/gos/common-voice/). Is this person a member of your team?
